### PR TITLE
Readonly role

### PR DESCRIPTION
**Pull request by odinuv**
- disable sync actions for readonly user
- add another check so as not to execute jobs if they pass api (e.g. through orchestrator)

https://keboola.atlassian.net/browse/PS-1422

### CODE REVIEW

**Comment by odinuv** [comment]
Mergnul bych to teda az bude UI ready (dneska to stejne mergnout nejde uz)

**Comment by ujovlado** [comment]
Kuknem.

File: Tests/Controller/ActionControllerTest.php [hunk 1]
```diff
@@ -330,6 +304,36 @@ public function testActionTest()
...
+        $container = self::$container;
+        $container->set("syrup.storage_api", $this->getStorageServiceStubDcaPython(false, 'readOnly'));
+        $container->get('request_stack')->push($request);
+
+        $ctrl = new ActionController();
+        $ctrl->setContainer(self::$container);
+        $ctrl->preExecute($request);
+        $this->expectException(UserException::class);
+        $this->expectExceptionMessage('As a readOnly user you cannot perform any actions.');
+        $ctrl->processAction($request);
```
Comment (ujovlado) [inline Tests/Controller/ActionControllerTest.php:319]: Cakal by som, ze tu sa tiez checkne getStatusCode a ci je to 400ka. Ale chapem, ze to spadne vnutri tak to nie je uplne jednoduche.

**Review by ujovlado — APPROVED** [review]
Vyzera to cajk.

**Comment by ujovlado** [comment]
> Mergnul bych to teda az bude UI ready (dneska to stejne mergnout nejde uz)

Pre istotu aj sem. Poslime to dalej, nemusime s tym cakat na UI.

**Pull request merged**